### ISSUE TITLE
Can`t merge Computers together

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#870]
**Reported by didi s 11 on 3 Nov 39380815 20:26 UTC**
```
Host 2886118: CPU not compatible (GenuineIntel Pentium III (Coppermine), GenuineIntel Pentium III (Coppermine)) 
Host 2981731: CPU not compatible (GenuineIntel Pentium III (Coppermine), GenuineIntel Pentium III (Coppermine)) 
No hosts are eligible for merging with this one. 
```

**Comment by romw** [comment]
**Commented by Nicolas on 30 Aug 39380834 20:26 UTC**
That message means you're trying to merge a host X with 2886118 and 2981731. We need to know what X is.

Is it [1709412](http://setiathome.berkeley.edu/show_host_detail.php?hostid=1709412)?

**Comment by romw** [comment]
**Commented by didi-s-11 on 22 Oct 39387444 09:46 UTC**
Folowing Comuters are the same:
ID: 1709412
ID: 2886118
ID: 2981731
but I can`t merge them together - CPU not compatible 
GenuineIntel Pentium III (Coppermine), GenuineIntel Pentium III (Coppermine)

**Comment by romw** [comment]
**Commented by boincadm on 6 Aug 39392250 04:26 UTC**
(In [18196]) - web: fix host-merge logic.  Fixes #893
